README.md

```diff
@@ -28,11 +28,6 @@ Please feel free to send a pull request for any other free and downloadable top
 - Download page: https://builtwith.com/top-1m
 - Download link: https://builtwith.com/dl/builtwith-top1m.zip
 
-### Statvoo
-
-- Download page: https://statvoo.com/top/ranked
-- Download link: https://statvoo.com/dl/top-1million-sites.csv.zip
-
```

Comment:
Hi @shaunpud,

Thanks for your contribution. I have some other ideas about the dead links. Would you please revise this pull request to add the new top 1M domain source and let me handle the other part? Thanks again!